hm15/hm15_2_files.py

```diff
@@ -0,0 +1,26 @@
+"""Homework 15 - 10. Datetime """
+
+import datetime
+
+
+def task_ten():
```

Comment:
Pass first_date_str and second_date_str to the function, so it give you more flexibility -> find diff for ANY dates
```suggestion
def task_ten(first_date_str, second_date_str):
```